### PR TITLE
Rename withCollectionView param to collectionView in IGListAdapterUpdaterDelegate.

### PR DESCRIPTION
**Pull request by Malecks**
## Changes in this pull request

Issue fixed: #466 

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [ ] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by zhubofei** [comment]
@Malecks Sorry, this one dups with #491

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ This release closes the [3.0.0 milestone](https://github.com/Instagram/IGListKit
...
+// OLD
+- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater 
+  didPerformBatchUpdates:(IGListBatchUpdateData *)updates 
+  withCollectionView:(UICollectionView *)collectionView;
+  
+// NEW
+- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater 
+  didPerformBatchUpdates:(IGListBatchUpdateData *)updates 
+  collectionView:(UICollectionView *)collectionView;
+```
```
Comment (rnystrom) [inline CHANGELOG.md]: Mentioned this in the other PR, but probably don't need all this, I _think_ Xcode will offer the right suggestion.

**Comment by rnystrom** [comment]
Looks perfect! #491 beat you to the punch and already landed internally, but this is great.

**Comment by Malecks** [comment]
@rnystrom Thanks for taking the time to have a look! Will find another issue to work on :)

**Pull request closed without merging**